.bazelrc

```diff
@@ -21,4 +21,6 @@ common --enable_runfiles
 common --registry=file:///%workspace%/misc/bazel/registry
 common --registry=https://bcr.bazel.build
 
+common --@rules_dotnet//dotnet/settings:strict_deps=false
```

Comment:
Yeah I'll add a comment.

The problem is this: https://github.com/bazelbuild/rules_dotnet/issues/413 `strict_deps` in C# also applies to 3rd-party deps, and when we pull in (for example) the `xunit` package, there's no code in this at all, it just depends transitively on its implementation packages. I didn't want to add direct dependencies on package internals, so rather turned strict deps off.

Reply:
ok, now this makes sense.

If this is just affecting xUnit, might it make sense to keep strict deps on, but wrap `@paket.main//xunit` into a `csharp_library` exporting `@paket.main//xunit.extensibility.core`? As you noted in the issue, strict deps have their advantages...

Reply:
Xunit was the first dependency I added, and I hit this problem, I didn't investigate the other dependencies. I'll have a look and test out how many more problems turning on strict deps would cause.

Reply:
I've added the comment now, but that doesn't mean I'll not investigate turning strict deps on again.

Reply:
Okay, I tried, and I think it's too hard. For example, the namespace `Microsoft.CodeAnalysis` isn't even defined in that nuget package, but rather in a transitive (not direct) dependency of it!
We can't expect to hunt down namespace, and symbol definitions in a big transitive dependency tree whenever you change some code, this is very user unfriendly.